extensions/ql-vscode/src/telemetry.ts

```diff
@@ -157,12 +160,18 @@ export class TelemetryListener extends ConfigListener {
       ? CommandCompletion.Cancelled
       : CommandCompletion.Failed;
 
+    const cliVersion = this._cli
+      ? (await this._cli.getVersion()).toString()
```

Comment:
Should we be catching the error here and setting the `cliVersion` to something like `errored`? It seems like we could start missing command usages if the CLI server is in a bad state.

Reply:
After the change, this should no longer be relevant.